README.md

```diff
@@ -40,9 +40,16 @@ The app should now be started and can be found at [http://localhost:5000](http:/
 
 We have added significant changes to the database, adding the 2023 data and some other data used in the web app.
 
-### Steps to Update Database
+### Create New Database from Dump
 
-To update the databse to have the needed extra data in order to run the run the app, as well as to update the database with 2023 player data:
+To create a new database with all the data needed to run the app, run the following commands in your terminal:
+1. Open your mysql interface from the project root folder: `mysql -u user -p`
```

Comment:
The project page specifies we need to use a MariaDB database. I may be misunderstanding this, but it seems this requires MySQL to be running rather than MariaDB.

Reply:
I'll fix the wording.